server/settings/test.py

```diff
@@ -4,7 +4,7 @@ class TestConfig(LocalConfig):
     ENV = 'test'
     DEBUG_TB_INTERCEPT_REDIRECTS = False
 
-    SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:@localhost:5432/oktest'
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///../database.db'
```

Comment:
This should be different from the dev database. The URI "sqlite://" will give you a temporary in-memory database: https://pythonhosted.org/Flask-Testing/#testing-with-sqlalchemy